admin_manual/configuration_server/caching_configuration.rst

```diff
@@ -0,0 +1,193 @@
+==========================
+Configuring Memory Caching
+==========================
+
+You can significantly improve your ownCloud server performance with memory 
+caching, where frequently-requested objects are stored in memory for faster 
+retrieval. There are two types of caches to use: PHP opcache, and data caching 
+for your Web server. If you do not install a memcache you will see a warning on 
+your ownCloud admin page. (A memcache is not required and you may ignore the warning if you prefer.)
+
+A PHP opcache stores compiled PHP scripts so they don't need to be re-compiled 
```

Comment:
We need then APC howto because a lot of our customers are still on PHP 5.4. Which will be the sad state of the world as long as RHEL 6 and CentOS 6 are still loose in the wild. It's really not that complicated. 

As long as config.sample.php mentions XCache, we need to say something about it, even if it's only "supported but icky, don't use."